File: app/src/main/java/br/com/imerys/imeryscomercialcatalog/domain/Solutions.java
package br.com.imerys.imeryscomercialcatalog.domain;

import java.util.HashMap;
import java.util.Map;
import javax.annotation.Generated;

@Generated("jsonschema2pojo")
public class Example {

    private Integer numero;
    private String mercado;
    private String categoria;
    private String imagem;
    private String nome;
    private String regiao;
    private String url;
    private String descricao;
    private Map<String, Object> additionalProperties = new HashMap<String, Object>();

    public Integer getNumero() {
        return numero;
    }

    public void setNumero(Integer numero) {
        this.numero = numero;
    }

    public String getMercado() {
        return mercado;
    }

    public void setMercado(String mercado) {
        this.mercado = mercado;
    }

    public String getCategoria() {
        return categoria;
    }

    public void setCategoria(String categoria) {
        this.categoria = categoria;
    }

    public String getImagem() {
        return imagem;
    }

    public void setImagem(String imagem) {
        this.imagem = imagem;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getRegiao() {
        return regiao;
    }

    public void setRegiao(String regiao) {
        this.regiao = regiao;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public String getDescricao() {
        return descricao;
    }

    public void setDescricao(String descricao) {
        this.descricao = descricao;
    }

    public Map<String, Object> getAdditionalProperties() {
        return this.additionalProperties;
    }

    public void setAdditionalProperty(String name, Object value) {
        this.additionalProperties.put(name, value);
    }

}

File: app/src/main/java/br/com/imerys/imeryscomercialcatalog/domain/Solutions.kt
package br.com.imerys.imeryscomercialcatalog.domain

data class Solutions (private var _id:String,
                      private var _markets:String,
                      private var _category:String,
                      private var _image:String,
                      private var _name:String,
                      private var _region:String,
                      private var _link:String,
                      private var _description:String,){
    var id: String = _id
        get() {
            return field
        }
        set(value) {
            field = value
        }
    var markets: String = _markets
        get() {
            return field
        }
        set(value) {
            field = value
        }
    var category: String = _category
        get() {
            return field
        }
        set(value) {
            field = value
        }
    var image: String = _image
        get() {
            return field
        }
        set(value) {
            field = value
        }
    var name: String = _name
        get() {
            return field
        }
        set(value) {
            field = value
        }
    var region: String = _region
        get() {
            return field
        }
        set(value) {
            field = value
        }
    var link: String = _link
        get() {
            return field
        }
        set(value) {
            field = value
        }
    var description: String = _description
        get() {
            return field
        }
        set(value) {
            field = value
        }
}